datajunction-clients/python/tests/examples/project1/roads/repair_orders_cube.cube.yaml

```diff
@@ -6,4 +6,4 @@ metrics:
 dimensions:
   - ${prefix}roads.hard_hat.state
   - ${prefix}roads.dispatcher.company_name
-filter: []
+filters: []
```

Comment:
Hey @samredai check this out... it turns out dataclasses do a better job in attribute validation than pydantic ;)

Reply:
Wow, look at that! 🤯😭